packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.scss

```diff
@@ -1,7 +1,6 @@
 .wallets-gradient {
     position: relative;
     display: inline-block;
-    border-radius: 0.8rem;
```

Comment:
Removed this border radius change as it's appearing on the wallet cashier header